src/southbridge/intel/common/spi.c

```diff
@@ -1110,29 +1110,76 @@ __weak void intel_southbridge_override_spi(struct intel_swseq_spi_config *spi_co
 {
 }
 
+#if CONFIG_SOUTHBRIDGE_INTEL_COMMON_SPI_SILVERMONT
+#define ILB_BASE_ADDRESS		0xfed08000
+#define SPI_BASE_ADDRESS		0xfed01000
```

Comment:
No, no, no. This is bad. Read it from the PCI registers of LPC bridge, like the datasheet indicates. No hardcoding.

Besides, SPI_BASE_ADDRESS can be obtained with `get_spi_bar`. Similar thing must be done with ILB base